autonomy/cli/mint.py

```diff
@@ -116,6 +120,7 @@ def mint(  # pylint: disable=too-many-arguments
 @dependencies_decorator
 @nft_decorator
 @owner_flag
+@update_flag
```

Comment:
A component can be updated using the same syntax as minting by adding --update TOKEN_ID flag 

> autonomy mint PACKAGE_TYPE PACKAGE_PATH ... --update TOKEN_ID